content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -93,6 +109,20 @@ the `startingCSV` field of the subscription is not possible when using skiprange
 The above warning means that skiprange by itself is useful for teams who are not interested in
 supporting directly installing versions within a given range.
 
+#### Example for SkipRange
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+    name: elasticsearch-operator.v4.1.2
+    namespace: placeholder
+    annotations:
+        olm.skipRange: '>=4.1.0 <4.1.2'
+```
+
+The `skipRange` parameter is used to ignore the multiple versions of an operator during the upgrade cycle. In the above example the operator version will be avoided from `4.1.0` to `4.1.1`.
```

Comment:
> In the above example the operator version will be avoided from `4.1.0` to `4.1.1`

This sounds a little odd. I would say something like:

> In the above example, `elasticsearch-operator.v4.1.2` will replace all operators in its package with versions falling in the range `>=4.1.0 <4.1.2`, while at the same time preventing those operators from being installed.

Reply:
I added the explanation. I agree with it as it clearly explains the use of skipRange.